style.css

```diff
@@ -390,8 +399,117 @@ h4 {
   color: #42526e;
 }
 
+#empty-footer {
+  display: none;
+}
+
 footer {
   display: flex;
   flex-direction: column;
   align-items: center;
 }
+
+@media only screen and (min-width: 768px) {
+  .navbar {
+    display: block;
+    padding: 26px;
+    color: #344563;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  h2 {
+    display: none;
+  }
+
+  .header {
+    display: none;
+  }
+
+  a.arrow2 {
+    display: none;
+  }
+
+  a.arrow {
+    border: 1px solid #dfe1e6;
+    border-radius: 50%;
+    width: 40px;
+    height: 40px;
+    margin-top: 40px;
+    padding-bottom: 20px;
+    padding-right: 24px;
+    margin-left: 48%;
+    display: flex;
+  }
+
+  .landing-page {
+    background-image: url("./images/header-background.svg");
+    width: 100%;
+    text-align: center;
+  }
+
+  .proj-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 24px 32px;
+    margin-top: 52px;
+  }
+
+  .about-me {
+    max-width: 100%;
+    background-image: url("./images/illustration-about-me-desktop.png");
+    background-repeat: no-repeat;
+  }
```

Comment:
- [ ] Please add the `About me` text header for your about section 

Your design 👇 
![Screen Shot 2022-06-09 at 23 03 44](https://user-images.githubusercontent.com/75126481/172944185-c1da879e-a444-44ea-a7c8-ef9da83ca35e.png)

Original Design 👇 

![Screen Shot 2022-06-09 at 23 06 04](https://user-images.githubusercontent.com/75126481/172944497-502eef7d-13c8-4dad-a6aa-e42ae3fb5fc6.png)

Reply:
Done